ups/eupspkg.cfg.sh

```diff
@@ -17,20 +17,20 @@ install()
     case $(uname -s) in
         Linux*)
             ana_platform="Linux-x86_64"
-            conda_packages="conda_packages-linux-64.txt"
+            conda_packages="conda3_packages-linux-64.txt"
             ;;
         Darwin*)
             ana_platform="MacOSX-x86_64"
-            conda_packages="conda_packages-osx-64.txt"
+            conda_packages="conda3_packages-osx-64.txt"
```

Comment:
Same, change in a text file outside the scope of this JIRA. But notable since it could break things.